src/scancode/templates/html-app/template.html

```diff
@@ -113,6 +113,7 @@
   <script src="{{ assets_dir }}/jquery.min.js"></script>
   <script src="{{ assets_dir }}/jstree.min.js"></script>
   <script src="{{ assets_dir }}/jquery.dataTables.min.js"></script>
+  <script type="text/javascript" src="{{ assets_dir }}/data.json"></script>
```

Comment:
To avoid same-origin or json vs. js issues, should the file have a .js extension instead of .json?